tests/testthat/test-select_helpers.R

```diff
@@ -40,6 +81,12 @@ test_that("select_helpers: tidy_select_variables", {
 
 test_that("select_helpers: tidy_plus_plus", {
   mod <- glm(response ~ age * trt + grade, gtsummary::trial, family = binomial)
+  mod2 <- glm(response ~ stage + grade * trt,
```

Comment:
add tests with this complex contrasts case

```
  mod <- glm(
    response ~ stage + grade + trt + factor(death),
    gtsummary::trial,
    family = binomial,
    contrasts = list(
      stage = contr.treatment(4, 3), grade = contr.treatment(3, 2),
      trt = contr.treatment(2, 2), "factor(death)" = matrix(c(-3, 2))
    )
  )
```

Reply:
✔️